lib/MetaCPAN/Types/Internal.pm

```diff
@@ -93,16 +95,27 @@ coerce Profile, from HashRef,
 subtype Tests,
     as Dict [ fail => Int, na => Int, pass => Int, unknown => Int ];
 
-subtype BugSummary,
+subtype BugsRT,
     as Dict [
     (
         map { $_ => Optional [Int] }
-            qw(new open stalled patched resolved rejected active closed)
+            qw( active closed new open patched rejected resolved stalled )
     ),
-    type   => Str,
     source => Str
     ];
 
+subtype BugsGithub,
```

Comment:
On second thought, these aren't all bugs.  GitHub or RT issues might be wishlist items, so calling everything a bug might be misleading.